tests/commands/test_runner.py

```diff
@@ -210,6 +210,10 @@ def test_global_trace_tags(self):
             out = subprocess.check_output(["ddtrace-run", "python", "tests/commands/ddtrace_run_global_tags.py"])
             assert out.startswith(b"Test success")
 
+    @pytest.mark.subprocess(env=dict(DD_TRACE_GLOBAL_TAGS="a:True,b:0,c:C"), err=None)
```

Comment:
Update the error parameter to match the expected warning

```suggestion
    @pytest.mark.subprocess(env=dict(DD_TRACE_GLOBAL_TAGS="a:True,b:0,c:C"), err=.....)
```